src/main/java/creativei/entity/Inquiry.java

```diff
@@ -74,8 +71,8 @@ public Inquiry(Long id){
     private AreaOfInterest areaOfInterest;
     @OneToOne(cascade = CascadeType.ALL)
     private InquiryAddress inquiryAddress;
-    @OneToMany(cascade = CascadeType.ALL,mappedBy = "inquiry")
-    private List<InquiryEducation> inquiryEducation=new ArrayList<>();
+    @OneToMany(cascade = CascadeType.ALL, mappedBy = "inquiry")
+    private List<InquiryEducation> inquiryEducations =new ArrayList<>();
```

Comment:
Avoid initializing here to avoid memory wastage

Reply:
null pointer exception in thrown if not initialized.